travis/install-tools.sh

```diff
@@ -1,19 +1,27 @@
 #!/bin/bash
 set -e -o pipefail
 
+CONDA_ENV="$(pwd)/tools/conda-tools/default"
+
 # Set to conda's java
-export JAVA_HOME="$(pwd)/tools/conda-tools/default/jre"
+export JAVA_HOME="$CONDA_ENV/jre"
 
 echo "Installing and validating bioinformatic tools"
 export CONDA_ENVS_PATH=tools/conda-cache:tools/conda-tools/default
 
 PYVER=`echo $TRAVIS_PYTHON_VERSION | cut -c 1`
 
-for i in $(seq 3); do
-  conda create --quiet -y -m -c broad-viral -c r -c bioconda -c conda-forge -c defaults -p tools/conda-tools/default --file requirements-conda.txt --file requirements-conda-tests.txt --file requirements-py$PYVER.txt python="$TRAVIS_PYTHON_VERSION" && break
-  sleep 5
-done
+if [ ! -d $CONDA_ENV ]; then
```

Comment:
An alternative to checking if the directory exists would be to pass `--clobber` to `conda create`, but it's probably best to check for the existence of the environment root as you're doing here.
```
  --clobber             Allow clobbering of overlapping file paths within
                        packages, and suppress related warnings.
```

Reply:
It wasn't totally clear to me whether `--clobber` would blow away the existing environment and start from scratch, or just add and overwrite the parts it was asked to... so I erred on the side of just doing nothing if it already existed.